### PR TITLE
New feature: Room announcement

### PR DESCRIPTION
**Pull request by billtt**
@RocketChat/core 

Users that have room editing privileges can set announcement for the room. Announcements will show on top of the room, just below the room name. Please see the attachment for how the announcement looks like.

image

> the message in the screenshot "Room topic changed to ..." has been fixed in latest commit (should be "Room announcement changed to...")

### CODE REVIEW

**Comment by Nemra1** [comment]
+1
i love it

File: packages/rocketchat-ui/client/views/app/room.html [hunk 1]
```diff
@@ -22,6 +22,11 @@
 						<span class="room-topic">{{{RocketChatMarkdown roomTopic}}}</span>
 					</h2>
 				</header>
+					{{#if showAnnouncement}}
+                        <div class="fixed-title announcement">
```
Comment (graywolf336) [inline packages/rocketchat-ui/client/views/app/room.html]: Can you fix this spacing and indentation?

**Comment by billtt** [comment]
@graywolf336 In our team it is required to always be there. Otherwise a simply @all would suffice. But I'll see if it's better to have the option to hide it when it's in use.

File: packages/rocketchat-ui/client/views/app/room.html (hunk 1 above)
Comment (billtt) [inline packages/rocketchat-ui/client/views/app/room.html]: fixed @graywolf336

**Pull request merged**